docs/getting-started/supported-components.md

```diff
@@ -0,0 +1,125 @@
+# Supported Components
+
+The following list of Material Design components and features highlights 
+those currently supported by Material-UI in bold.
+
+While we strive to follow the guidelines where practical (applying 
+common sense where guidelines contadict - a more common occurence than 
+one might expect), we do not expect to support every component, or every 
+feature of every component, but rather to provide the building blocks to 
+allow users to create compelling user interfaces and experiences.
+
+If you wish to add support for a component or feature not highlighted 
+here, please search for the relevant [GitHub Issue](https://github.com/callemall/material-ui/issues), or create a new one 
+to discuss the approach before submitting a PR.
+
+- **[App Bar](https://material.io/guidelines/layout/structure.html#structure-app-bar)**
+- [App shortcut icons](https://material.io/guidelines/style/icons.html#icons-app-shortcut-icons)
+  - Icon
+  - Avatar
+  - Group avatar
+- **[Bottom navigation](https://www.google.com/design/spec/components/bottom-navigation.html)**
+- [Bottom sheets](https://www.google.com/design/spec/components/bottom-sheets.html)
+- **[Buttons](http://www.material-ui.com/#/components/buttons)**
+  - **[Flat & raised buttons](https://www.google.com/design/spec/components/buttons.html#buttons-flat-raised-buttons)**
+  - [Toggle buttons](https://www.google.com/design/spec/components/buttons.html#buttons-toggle-buttons)
+  - **[Icon toggles](https://www.google.com/design/spec/components/buttons.html#buttons-toggle-buttons) (Custom Checkbox)**
+  - [Dropdown: Mobile: Dropdown buttons](https://www.google.com/design/spec/components/buttons.html#buttons-dropdown-buttons) (DropDownMenu)
+  - [Dropdown: Mobile: Generic overflow](https://www.google.com/design/spec/components/buttons.html#buttons-dropdown-buttons)
+  - [Dropdown: Mobile: Segmented](https://www.google.com/design/spec/components/buttons.html#buttons-dropdown-buttons)
+  - [Dropdown: Mobile: Editable segmented](https://www.google.com/design/spec/components/buttons.html#buttons-dropdown-buttons)
+  - [Dropdown: Desktop: Dropdown buttons](https://www.google.com/design/spec/components/buttons.html#buttons-dropdown-buttons)
+- **[Buttons: Floating Action Button](https://www.google.com/design/spec/components/buttons-floating-action-button.html)**
+  - **[Trigger](https://www.google.com/design/spec/components/buttons-floating-action-button.html#buttons-floating-action-button-transitions)**
+  - [Toolbar](https://www.google.com/design/spec/components/buttons-floating-action-button.html#buttons-floating-action-button-transitions)
+  - [Speed dial](https://www.google.com/design/spec/components/buttons-floating-action-button.html#buttons-floating-action-button-transitions) - #2851 
```

Comment:
I would avoid linking issues in the docs. I think that it would takes too much effort keeping it up to date.
At least, this is not a real link once rendered by the documentation.